documentation/src/main/resources/pages/ditto/httpapi-protocol-bindings-cloudevents.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Cloud Events HTTP protocol binding
+keywords: binding, protocol, http, cloudevents
+tags: [binding, protocol, http]
+permalink: httpapi-protocol-bindings-cloudevents.html
+---
+
+Implements the [HTTP Protocol Binding for CloudEvents - Version 1.0](https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md).
+
+Unless mentioned otherwise, the endpoint following the Cloud Events specification for the HTTP binding in version 1.0.
+
+## Cloud Events features
+
+The Cloud Events endpoint provides an alternative to the other connectivity APIs to stream data into your instance.
+
+## Cloud Events endpoint
+
+The Cloud Events endpoint is accessible at these URLs (depending on which API version to use):
+
+```
+http://localhost:8080/api/1/cloudevents
+http://localhost:8080/api/2/cloudevents
+```
+
+### Authentication
+
+A user who connects to the Cloud Events endpoint can be authenticated by using
+
+* HTTP BASIC Authentication by providing a username and the password of a user managed within nginx or
+* a JSON Web Token (JWT) issued by an OpenID connect provider.
+
+See [Authenticate](basic-auth.html) for more details.
+
+## Cloud Events protocol format
+
+The source must be a Cloud Event, encoded in the format for the HTTP binding. It may be encoded in *binary content mode*
+or in *structural content mode*.
+
+The *data content type* of the event must be `application/json`.
+
+The *data schema* must start with `ditto:`, for example `ditto:org.eclipse.vorto:TestDevice:1.0.0`.
+
+The events *payload* must in the Ditto Protocol JSON format as defined in the
+[Protocol specification](protocol-specification.html).
+
+## Publishing events to the endpoint
+
+Publishing events to the endpoint can be done by directly sending HTTP requests, conforming to the Cloud Events
+HTTP binding specification. Or by using other technologies that have adopted Cloud Events.
+
+### Knative eventing
+
+The endpoint can directly be configured as a [Knative eventing](https://knative.dev/docs/eventing/) destination.
+
+In the following example, a Knative eventing flow is configured to normalize the payload with a Vorto converter
+and send the result to Ditto's cloud events endpoint:
+
+~~~yaml
+apiVersion: flows.knative.dev/v1
+kind: Sequence
+metadata:
+ name: digital-twin
+spec:
+  channelTemplate:
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: KafkaChannel
+    spec:
+      numPartitions: 1
+      replicationFactor: 1
+  steps:
+  - ref:
+      # Convert incoming payload to the Ditto format
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: vorto-converter
+      namespace: drogue-iot
+  reply:
+    # Deliver to Ditto Cloud Events endpoint
+    uri: http://ditto:ditto@ditto-nginx.drogue-iot.svc.cluster.local:8080/api/2/cloudevents
```

Comment:
Are you sure that you want `....drogue-iot...` in here? ;)

Reply:
True.